server/loader.py

```diff
@@ -168,8 +55,27 @@ def validate_settings(event):
 
 
 def load(info):
+    # Load the mako template for Minerva and serve it as the root document.
+    minerva_mako = os.path.join(os.path.dirname(__file__), "minerva.mako")
+    minerva_webroot = Webroot(minerva_mako)
+
+    # Load resources from all enabled plugins, which allows Minerva
+    # to be extended by other Girder plugins.
+    plugins = ModelImporter.model('setting').get(SettingKey.PLUGINS_ENABLED,
+                                                 ())
+    plugins = list(plugins)
+    if 'minerva' in plugins:
+        plugins.remove('minerva')
+    vars = {
+        'plugins': plugins,
+        'apiRoot': '/api/v1',
+        'staticRoot': '/static',
+        'title': 'Minerva'
+    }
+    minerva_webroot.updateHtmlVars(vars)
```

Comment:
> I think you are talking about "plugin loading" meaning on the server side, whereas I'm only intending this to have an effect on the client side resources of the loaded plugins, i.e. to have them be served in the minerva mako template.

Correct, but more importantly I don't think there is any way that this plugin list can be different from the list that girder is [already initializing](https://github.com/girder/girder/blob/master/girder/utility/server.py#L103).  My code above uses the list girder is using for it's own web root (as well as the static and api roots).

> Will do. Can you give an example of such a scenario so I can understand the issue better?

http://girder.readthedocs.org/en/latest/deploy.html#reverse-proxy